ui/component/IframeReact/view.jsx

```diff
@@ -0,0 +1,35 @@
+// @flow
+import React from 'react';
+
+type Props = {
+  fullHeight: boolean,
+  src: string,
+  title: string,
+};
+
+export default function I18nMessage(props: Props) {
```

Comment:
This doesn't do anything. If iframes are serving files from the same domain, growable iframes would work. Ironically we just changed this.